packages/block-library/src/audio/edit.js

```diff
@@ -118,24 +108,19 @@ class AudioEdit extends Component {
 	render() {
 		const { autoplay, caption, loop, preload, src } = this.props.attributes;
 		const { setAttributes, isSelected, className, noticeUI } = this.props;
-		const { editing } = this.state;
-		const switchToEditing = () => {
-			this.setState( { editing: true } );
-		};
 		const onSelectAudio = ( media ) => {
 			if ( ! media || ! media.url ) {
 				// in this case there was an error and we should continue in the editing state
 				// previous attributes should be removed because they may be temporary blob urls
 				setAttributes( { src: undefined, id: undefined } );
-				switchToEditing();
 				return;
 			}
 			// sets the block's attribute and updates the edit component from the
 			// selected media, then switches off the editing UI
 			setAttributes( { src: media.url, id: media.id } );
 			this.setState( { src: media.url, editing: false } );
```

Comment:
We removed the `editing` state in this PR, the state `src` was removed in #6820, so we could delete this line. 
```suggestion
```

Reply:
Good catch, I removed the line in [d7681fa](https://github.com/WordPress/gutenberg/pull/19158/commits/d7681fa9768deef4eabc3a3dbbd9cbe29b482aad).